### PR TITLE
use codecov.io for code coverage

### PR DESCRIPTION
**Pull request by fwbrasil**
Codacy isn't reliable for code coverage reports. I've seen it report more than 100% coverage more than once.

cc/ @williamboxhall @stevenheidel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,10 @@
 =========================
 A library for expressive and efficient service composition
 
-[![Build Status](https://secure.travis-ci.org/getclump/clump.png)](https://api.travis-ci.org/getclump/clump.svg?branch=master)
-[![Codacy Badge](https://www.codacy.com/project/badge/5fd8030f7c494b739f6ec5c1963371a8)](https://www.codacy.com/public/fwbrasil/clump)
-[![Join the chat at https://gitter.im/getclump/clump](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/getclump/clump?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://img.shields.io/travis/getclump/clump.svg)](https://api.travis-ci.org/getclump/clump.svg?branch=master)
```
Comment (fwbrasil) [inline README.md]: I also took the opportunity to use shields.io to make the badges more uniform.

File: project/Build.scala [hunk 2]
```diff
@@ -32,6 +33,8 @@ object Build extends Build {
       else
         Some("releases"  at nexus + "service/local/staging/deploy/maven2")
     },
+    ScoverageSbtPlugin.ScoverageKeys.coverageMinimum := 100,
+    ScoverageSbtPlugin.ScoverageKeys.coverageFailOnMinimum := false,
```
Comment (fwbrasil) [inline project/Build.scala]: Master doesn't have 100% coverage right now. After fixing it, we can change this flat to `true`.

**Comment by stevenheidel** [comment]
:+1:

**Comment by williamboxhall** [comment]
:+1:

**Pull request merged**